qa/acceptance-tests/src/test/java/io/camunda/it/auth/UserSearchingAuthorizationIT.java

```diff
@@ -77,60 +63,48 @@ class UserSearchingAuthorizationIT {
           DEFAULT_PASSWORD,
           List.of(new Permissions(ResourceType.USER, PermissionType.READ, List.of("*"))));
 
-  @AutoClose private static final HttpClient HTTP_CLIENT = HttpClient.newHttpClient();
-
   @BeforeAll
   static void setUp(@Authenticated(ADMIN) final CamundaClient adminClient) {
     createUser(adminClient, "user1");
     createUser(adminClient, "user2");
-    final var expectedCount = 6; // demo, admin, user1, user2, restricted, restrictedWithRead
-    waitForUsersToBeCreated(
-        adminClient.getConfiguration().getRestAddress().toString(), ADMIN, expectedCount);
-  }
 
-  private static UserSearchResponse searchUsers(final String restAddress, final String username)
-      throws URISyntaxException, IOException, InterruptedException {
-    final var encodedCredentials =
-        Base64.getEncoder()
-            .encodeToString("%s:%s".formatted(username, DEFAULT_PASSWORD).getBytes());
-    final HttpRequest request =
-        HttpRequest.newBuilder()
-            .uri(new URI("%s%s".formatted(restAddress, USER_SEARCH_ENDPOINT)))
-            .POST(HttpRequest.BodyPublishers.ofString(""))
-            .header("Authorization", "Basic %s".formatted(encodedCredentials))
-            .build();
-
-    final HttpResponse<String> response =
-        HTTP_CLIENT.send(request, HttpResponse.BodyHandlers.ofString());
-
-    return OBJECT_MAPPER.readValue(response.body(), UserSearchResponse.class);
+    Awaitility.await("should create users and import in ES")
+        .atMost(AWAIT_TIMEOUT)
+        .ignoreExceptions()
+        .untilAsserted(
+            () -> {
+              final var userSearchResponse = adminClient.newUsersSearchRequest().send().join();
+              assertThat(
+                      userSearchResponse.items().stream()
+                          .map(io.camunda.client.api.search.response.User::getUsername)
```

Comment:
🤔 We should probably rename the qa-util `User` to `TestUser` I did the same with roles and groups as I didn't like dealing with full import paths.